lib/modules/datasource/custom/readme.md

```diff
@@ -310,3 +310,21 @@ And the following custom manager:
   ]
 }
 ```
+
+Or if you have the datasource locally, you can also use a local registry with the `isLocalRegistry` option:
+
+```json
+{
+  "customDatasources": {
+    "local_generic": {
+      "isLocalRegistry": true,
+      "defaultRegistryUrlTemplate": "dependencies/{{packageName}}/versiontracker.json",
```

Comment:
To be fair, I'm not 100% sure if this would be a correct API, maybe this shouldn't say `registryUrl` but `registryPath` ?